### ISSUE TITLE
If you choose to overlap borders, then opacity will put in evidence the overlap.

### ISSUE THREAD

**Issue by arypbatista** [arypbatista/godot-bordered-polygon-2d#29]
If you choose to overlap borders, then opacity will put in evidence the overlap.

![image](https://cloud.githubusercontent.com/assets/4958871/23332929/babb2020-fb60-11e6-845c-bb850aaa9e66.png)

**Comment by jtaart** [comment]
Isn't this the reason you want to have overlap?

**Comment by arypbatista** [comment]
No, I want to use overlap to hide fill borders, plus have some texture-alpha opacity to blend with fill texture.

I would like this to behave differently, to first merge textures and then apply opacity to the whole node, so you will see opacity but you won't be able to detect where the inner polygon ends.

But unless we split border polygons in two (outer border, inner border) and apply opacity differently to what is outside and inside, I think this might only be achieved throw _draw.
